src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.random.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i >= 3) {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].draw();
+                continue;
+            }
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
+    }
```

Comment:
if you will need to change length of array from 6 to other value, you will need to make to changes to this code so it can keep working as expected - replace 6 and 3, that's why we should introduce a constant and depend on it instead of hardcoding values, Also, better write if-else here, and put draw call outside of it (common for both cases)